### DISCUSSION TITLE
show / hide third part resources

### DISCUSSION THREAD

**Discussion by juangdnar** [filamentphp/filament#10393] [Help] [answered]
How can I show or hide options from the main menu, from resources that I have installed through the plugins, for example I am using the **activity log** plugin and I don't know how to condition it so that it is shown only to administrator-type users. annex example image; Thanks in advance.

![image](https://github.com/filamentphp/filament/assets/115037170/3fee6c36-152e-4850-ba0f-77a41e8f58e9)

**Comment by leandrocfe** [comment]
If it is a resource, you should use [policies](https://filamentphp.com/docs/3.x/panels/resources/getting-started#authorization).
You should also check the plugin page to get some information about the Authorization

**Reply by juangdnar** [reply]
Where are those third-party resources hosted? They are not in the app/filament/resources path like the rest

**Reply by leandrocfe** [reply]
Did you check some information about authorization in the plugin page?

**Reply by juangdnar** [reply]
Yes, I am reviewing it but they do not mention how to apply spatie roles and permissions, they allow publishing the configuration file, but they do not specify how to restrict access through the roles. This is the pluggin im using:

https://filamentphp.com/plugins/noxoua-activity-log

**Reply by leandrocfe** [reply, marked as answer]
It is a page. You should use https://filamentphp.com/docs/3.x/panels/pages#conditionally-hiding-pages-in-navigation

**Reply by juangdnar** [reply]
What I don't know is where that page is located to be able to add the Conditionally hiding pages in navigation

**Reply by leandrocfe** [reply]
Did you create this page?
https://activity-log.noxo.app/get-started/create-activity-page

**Reply by juangdnar** [reply]
I already found it, it was in Filament/Pages/Activities, thank you very much for the support. :)
